packages/frontend/components/Providence/CreateRecord.vue

```diff
@@ -26,7 +26,7 @@
 </template>
 
 <script lang="ts">
-import { postProvenance } from '~/services/azureFuncs';
+    import { postProvenance,getProvenance } from '~/services/azureFuncs';
```

Comment:
Formatting: Add a space after the comma.

Reply:
Fixed.